test/integration/master/synthetic_master_test.go

```diff
@@ -225,6 +225,72 @@ func TestStatus(t *testing.T) {
 	}
 }
 
+var expected1 = `etcdserver: request is too large`
+var expected2 = `rpc error: code = ResourceExhausted desc = grpc: trying to send message larger than max (3489913 vs. 2097152)`
+
+func TestObjectSizeResponses(t *testing.T) {
+
+	_, s, closeFn := framework.RunAMaster(nil)
+	defer closeFn()
+	transport := http.DefaultTransport
+
+	content1, err := ioutil.ReadFile("dummyfile1.txt")
```

Comment:
I guess instead of having these large files, you could add the labels through a loop.

Reply:
@ayushpateria yeah in the first place only I used a loop to get this file, will do that here as well. Thanks for the review, feel free to suggest other ways in which we can create a big object.